internal/accounts/contract-add.go

```diff
@@ -67,7 +71,21 @@ func addContract(
 		return nil, err
 	}
 
-	account, err := services.Accounts.AddContract(to, name, code, false)
+	if addContractFlags.ArgsJSON != "" {
```

Comment:
follow the syntax here https://github.com/onflow/flow-cli/blob/master/internal/transactions/send.go#L84-L93

Reply:
addressed